modules/signing_scriptworker/templates/dep-passwords.json.erb

```diff
@@ -18,6 +18,7 @@
         ["mac-v2-signing11.srv.releng.mdc2.mozilla.com:9110", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_dep_password"]) %>", ["macapp"], "signing_server"],
         ["mac-v2-signing12.srv.releng.mdc2.mozilla.com:9110", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_dep_password"]) %>", ["macapp"], "signing_server"],
         ["mac-v2-signing13.srv.releng.mdc2.mozilla.com:9110", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_dep_password"]) %>", ["macapp"], "signing_server"],
+        ["https://autograph-2.stage.mozaws.net", "<%= scope.function_secret(["autograph_fennec_dep_username"]) %>", "<%= scope.function_secret(["autograph_fennec_dep_password"]) %>", ["autograph_fennec"], "autograph"]
         ["https://autograph-external.prod.autograph.services.mozaws.net", "<%= scope.function_secret(["autograph_mar_dep_username"]) %>", "<%= scope.function_secret(["autograph_mar_dep_password"]) %>", ["autograph_mar384", "autograph_hash_only_mar384"], "autograph"]
```

Comment:
@g-k I'm confused a bit, please bear with me. You mentioned that the `mar dep` key is on the staging, yet here it's defined to access the production instance (line 22 in `dep-passwords.json.erb`) so if I understand this right, right now the dep-mar signing-scriptworkers will talk to production instance of Autograph. If that's the case, I see two possible scenarios:

a) we standardize that all dep keys should lie within the staging instance of Autograph, in which case we'll have to move the current `mar dep` key too
b) we standardize that all dep keys lie within the production instance of Autograph, in which case we'll have to move current `fennec dep` key there too

Does this make sense or have I misunderstood something?

Reply:
@MihaiTabara yeah, my mistake. I just checked the config and the MAR dep key is in prod.

Reply:
@g-k no problem, thanks for clarifications! Since we already have `mar dep` on production Autograph, let's keep that as a standard. So let's put all dep keys in the production Autograph instance and not touch staging. 

Do you need to generate another set of credentials or can you transplant those to prod?

Reply:
My thinking was, we often want to run staging releases, and we use dep keys in integration (though not for mar signing). Since I assume autograph-stage doesn't have the same expectations around uptime as autograph-prod, let's use autograph-prod for dep signing in production.

Reply:
Switched back to prod instance here. Once the Autograph deployment moves the configs from staging to production, we're green light to test here.

Reply:
@g-k deployed this today, proceeding with merging this and testing the dep-signing Fennec.